### PR TITLE
feat: front of community page

### PR DESCRIPTION
**Pull request by No3ml3**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [x] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [x] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [x] My `Javascript` generate no new console errors
- [x] I tested my code cross browsers
- [x] My slice is pixel perfect for both desktop and mobile according to design
- [x] I conducted basic QA to assure all features are working
- [x] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by jaalazard — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]

File: packages/frontend/web/src/pages/AllProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  id: number;
+  pseudo: string;
+  image: string;
+  color: string;
+  cocktail_count: number;
+  average_rating: number | null;
+}
+
+export default function AllProfile() {
+  const url = `${import.meta.env.VITE_API_URL}/user`;
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/AllProfile.tsx]: Pourquoi c'est dans le component ?

File: packages/frontend/web/src/pages/AllProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+interface UserProfile {
+  id: number;
+  pseudo: string;
+  image: string;
+  color: string;
+  cocktail_count: number;
+  average_rating: number | null;
+}
+
+export default function AllProfile() {
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/AllProfile.tsx]: Comment ça `AllProfile` ?

File: packages/frontend/web/src/router.tsx [hunk 3]
```diff
@@ -22,6 +23,10 @@ const router = createBrowserRouter([
     path: 'shaker',
     element: <AddCocktail />,
   },
+  {
+    path: 'community',
+    element: <AllProfile />,
```
Comment (YDrogen) [inline packages/frontend/web/src/router.tsx]: Pourquoi ne pas appeler le component `CommunityPage` par exemple ??

File: packages/frontend/web/tailwind.config.js [hunk 4]
```diff
@@ -154,6 +164,16 @@ export default {
...
+        'card-virgin-light-blue': '#A2C3F5',
+        'card-virgin-strong-pink': '#FB86DA',
+        'card-virgin-strong-purple': '#E08AEC',
+        'profil-picture-blue': '#07BCD5',
+        'profil-picture-green': '#35B888',
+        'profil-picture-orange': '#F2682F',
+        'profil-picture-yellow': '#F2DF2F',
+        'profil-picture-red': '#ED1E1E',
+        'profil-picture-pink': '#DD1D5E',
+        'profil-picture-purple': '#9F13AF',
```
Comment (YDrogen) [inline packages/frontend/web/tailwind.config.js]: Sympa les noms mi français mi anglais

**Review by YDrogen — CHANGES_REQUESTED** [review]
Pourquoi il y a des `.png` puis des `.webp` ?

**Review by jaalazard — DISMISSED** [review]
C'est beau

**Review by Cardamced — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Review by YDrogen — CHANGES_REQUESTED** [review]
.

**Review by jaalazard — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**